node/node.go

```diff
@@ -659,7 +659,7 @@ func (node *Node) Start() error {
 						errChan <- withError{
 							errors.WithStack(errConsensusMessageOnUnexpectedTopic), msg,
 						}
-						return libp2p_pubsub.ValidationReject
+						return libp2p_pubsub.ValidationIgnore
```

Comment:
why? did you spot any issue on the network? how many invalid_bound metrics you found on mainnet?

Reply:
Ah, I see, my bad. So side chain's `node.Start` does not handle beacon chain message. I see. Closing